tests/core/test_macros.py

```diff
@@ -9,12 +9,12 @@
 def filter_country(
     evaluator: MacroEvaluator, expression: exp.Condition, country: str
 ) -> exp.Condition:
-    return exp.and_(expression, f"country = '{country}'")
+    return exp.and_(expression, exp.column("country").eq(country))
 
 
 @macro("UPPER")
-def upper_case(evaluator: MacroEvaluator, expression: exp.Condition, country: str) -> exp.Condition:
-    return exp.true()
+def upper_case(evaluator: MacroEvaluator, expression: exp.Condition) -> str:
+    return f"UPPER({expression} + 1)"
```

Comment:
why `+ 1`?

Reply:
Looking at the test cases, looks like just to test and not a copy and paste error.

Reply:
yep, it's just a random test case